application/classes/Ushahidi/Validator/Post/Create.php

```diff
@@ -227,6 +227,8 @@ public function checkTags(Validation $validation, $tags)
 
 	public function checkValues(Validation $validation, $attributes, $fullData)
 	{
+
+		$attributes = !empty($fullData['values']) ? $fullData['values'] : [];
```

Comment:
I can see why the other changes are needed but I'm unsure why we'd recheck `values` unless it changes.